noxfiles/test_setup_nox.py

```diff
@@ -153,12 +153,15 @@ def pytest_ops(session: Session, mark: str, coverage_arg: str) -> None:
             "VAULT_NAMESPACE",
             "-e",
             "VAULT_TOKEN",
+            "-e",
+            "FIDES__DEV_MODE=false",
             CI_ARGS_EXEC,
             CONTAINER_NAME,
             "pytest",
             coverage_arg,
             OPS_TEST_DIR,
             "-m",
             "integration_saas",
+            "--tb=no",
```

Comment:
This is the flag that disables tracebacks, this means that no arguments or logs will be displayed during test failures.